.github/workflows/build.yml

```diff
@@ -98,6 +98,11 @@ jobs:
       - name: Build with Gradle
         run: ./gradlew build -PskipITs -PjdkVersion=${{ matrix.java-version }} -PskipDockerTests=false -x :clients:client-python:build
 
+      - name: Build spark connector with scala 2.13
+        run: |
+          ./gradlew -PscalaVersion=2.13 :spark-connector:spark-3.4:test
```

Comment:
Could you use the command below to skip ITs?
```
 ./gradlew  :spark-connector:spark-3.4:test -PskipITs -PjdkVersion=${{ matrix.java-version }} -PskipDockerTests=false -PscalaVersion=2.13 
 ./gradlew  :spark-connector:spark-3.5:test -PskipITs -PjdkVersion=${{ matrix.java-version }} -PskipDockerTests=false -PscalaVersion=2.13 

```

Reply:
the `skipDockerTests` param is true by default.